eng/versioning/alternative_external_dependency_versions.json

```diff
@@ -0,0 +1,55 @@
+{
+  "jackson_2.10":
+  [
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.10.0" },
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.10.0" },
+    { "com.fasterxml.jackson.core:jackson-core": "2.10.0" },
+    { "com.fasterxml.jackson.core:jackson-databind": "2.10.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-avro": "2.10.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-xml": "2.10.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jdk8": "2.10.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jsr310": "2.10.0" },
+    { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.10.0" },
+    { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.10.0" }
+  ],
+  "jackson_2.11":
+  [
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.11.0" },
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.11.0" },
+    { "com.fasterxml.jackson.core:jackson-core": "2.11.0" },
+    { "com.fasterxml.jackson.core:jackson-databind": "2.11.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-avro": "2.11.0"},
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-xml": "2.11.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jdk8": "2.11.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jsr310": "2.11.0" },
+    { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.11.0" },
+    { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.11.0" }
+  ],
+  "jackson_2.12":
+  [
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.12.0" },
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.12.0" },
+    { "com.fasterxml.jackson.core:jackson-core": "2.12.0" },
+    { "com.fasterxml.jackson.core:jackson-databind": "2.12.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-avro": "2.12.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-xml": "2.12.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jdk8": "2.12.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jsr310": "2.12.0" },
+    { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.12.0" },
+    { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.12.0" }
+  ],
+  "reactor_3.5":
+  [
+    { "io.projectreactor.netty:reactor-netty": "1.1.0-M5" },
+    { "io.projectreactor.netty:reactor-netty-http": "1.1.0-M5" },
+    { "io.projectreactor:reactor-core": "3.5.0-M5" },
+    { "io.projectreactor:reactor-test": "3.5.0-M5" }
+  ],
+  "reactor_3.4":
```

Comment:
If we're going to include Reactor 3.4 I'd also include Jackson 2.13 with baseline versions

Reply:
reactor is special here. They don't follow semver and add new APIs in minor versions, so I wanted to test 3.4.0 explicitly for things like this https://github.com/Azure/azure-sdk-for-java/pull/29806

Jackson doesn't have this problem and the tests we have for the latest 2.13.X are good enough.